src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -882,12 +934,57 @@ public LocalDate fromString(String string) {
     tableViewCandidates.getSelectionModel().setSelectionMode(SelectionMode.MULTIPLE);
     tableViewCandidates.setEditable(true);
 
+    clearAndDisableWinningRuleFields();
     choiceTiebreakMode.getItems().addAll(TieBreakMode.values());
     choiceTiebreakMode.getItems().remove(TieBreakMode.MODE_UNKNOWN);
+    choiceTiebreakMode.setOnAction(event -> {
+      clearAndDisableTiebreakFields();
+      String mode = getChoiceElse(choiceTiebreakMode, TieBreakMode.MODE_UNKNOWN);
+      switch (mode) {
+        case "Random", "Previous round counts (then random)", "Generate permutation" -> textFieldRandomSeed
+            .setDisable(false);
+      }
+    });
     choiceOvervoteRule.getItems().addAll(OvervoteRule.values());
     choiceOvervoteRule.getItems().remove(OvervoteRule.RULE_UNKNOWN);
     choiceWinnerElectionMode.getItems().addAll(WinnerElectionMode.values());
     choiceWinnerElectionMode.getItems().remove(WinnerElectionMode.MODE_UNKNOWN);
+    choiceWinnerElectionMode.setOnAction(event -> {
+      clearAndDisableWinningRuleFields();
+      setWinningRulesDefaultValues();
+      String mode = getChoiceElse(choiceWinnerElectionMode, WinnerElectionMode.MODE_UNKNOWN);
+      switch (mode) {
```

Comment:
Would it be preferable to switch on the actual enum and avoid using these hard-coded constants here?  I realize this would mean updating getChoiceElse, but this feels icky.